Foxtrot/Tests/RewriterTests.cs

```diff
@@ -67,6 +67,19 @@ public void Roslyn_CompatibilityCheck()
 
         [DeploymentItem("Foxtrot\\Tests\\TestInputs.xml"), DataSource("Microsoft.VisualStudio.TestTools.DataSource.XML", "|DataDirectory|\\TestInputs.xml", "RoslynCompatibility", DataAccessMethod.Sequential)]
         [TestMethod]
+        [TestCategory("Runtime"), TestCategory("CoreTest"), TestCategory("V4.5")]
+        public void Roslyn_TestTheSameCasesWithOldCompiler()
```

Comment:
.NET 4.5 means that this is a VS2013 compiler. Renamed to `TestTheRoslynCompatibilityCasesWithVS2013Compiler`.